frontend/public/components/custom-resource-definition.tsx

```diff
@@ -231,7 +231,7 @@ const Details: React.FC<{ obj: CustomResourceDefinitionKind }> = ({ obj: crd })
                   <Established crd={crd} />
                 </dd>
                 <DetailsItem label="Group" obj={crd} path="spec.group" />
-                <DetailsItem label="Version" obj={crd} path="spec.version" />
+                <DetailsItem label="Version" obj={crd} path="spec.versions[0].name" />
```

Comment:
I don't think we can assume the list is already sorted. We probably have to call `getLatestVersionForCRD`. We probably want `useMemo` to avoid resorting each array on every render.

Reply:
I think this can be `<DetailsItem label="Version" obj={crd} path={getLatestVersionForCRD(crd)} />` since it's the details page and not the list?  We may want `useMemo` in `<TableData className={tableColumnClasses[2]}>{getLatestVersionForCRD(crd)}</TableData>` on line 205?

Reply:
Yeah, the API says this field is _**used**_ to sort versions, but not that it _**is**_ sorted

Reply:
> I think this can be `<DetailsItem label="Version" obj={crd} path={getLatestVersionForCRD(crd)} />` since it's the details page and not the list? We may want `useMemo` in `<TableData className={tableColumnClasses[2]}>{getLatestVersionForCRD(crd)}</TableData>` on line 205?

`<DetailsItem label="Version" obj={crd} path={getLatestVersionForCRD(crd)} />` won't work as expected. The `path` is a path string which points to the object property we want to display. We can omit path and just do `<DetailsItem label="Latest Version" obj={crd}>{getLatestVersionForCRD(crd)}</DetailsItem>`